README.md

```diff
@@ -577,20 +443,234 @@ anything. If a company exits at a very high valuation relative to what was expec
+seem valuable because of high expectations that it can make future profit or be acquired.
+If a company like this takes money from investors, the investors determine the price they
+pay based on these educated guesses.
+In short, when considering early stage private companies like startups, there is more
+guesswork and more uncertainty about the future value of equity.
+
+## How Equity is Granted
+
+In this section we’ll lay out how equity is granted in practice, including the
+differences, benefits, and drawbacks of common types of equity.
+We explain restricted stock awards, stock options, and restricted stock units (RSUs),
+which are the most common forms of equity compensation, as well as a few less common ones.
+While the intent of each kind of equity grant is similar, they differ in many ways,
+particularly around taxation, as [covered later](#taxes-on-equity-compensation).
+
+The type of equity you get depends on the company, and in some (relatively rare) cases can
+be negotiated, but in general, RSUs are more common for larger companies and options more
 common for startups.
 
+### Restricted stock awards
+
+The most obvious approach to equity compensation would be for the company just to give
+someone stock.
+In practice, it turns out a company will only want to do this in certain situations, and
+even so, this is actually one of the least common ways to get equity.
+But we mention it first, since it’s the simplest form of equity compensation.
+
+- 🄳 A **restricted stock award** is when a company grants someone stock as a form of
+  compensation. The stock awarded has additional conditions on it so is called **restricted
+  stock**. *Restricted* here refers to the fact that the stock will be subject to repurchase
+  at a lower rate than fair market value or cost, and the repurchase right lapses over the
+  service-based vesting period.
+  - 🚧 Clarify legalese here.
+  - 🚧 Call out confusion of restricted stock units vs restricted stock awards.
+- Typically, stock awards are limited to executives or very early hires, since once the
+  value of the shares increases, the tax burden of receiving them can be too great for most
+  people. Usually, instead of restricted stock, an employee will get stock options.
+
 ### Stock options
 
-🄳 **[Employee stock options](https://en.wikipedia.org/wiki/Employee_stock_option) (or stock
-options, or ESOs)** are contracts that allow you to buy shares in the company you work for.
-When you buy shares, you are **exercising** the options.
+The most common way early-stage companies grant equity is through stock options.
+They get pretty complicated, and much of this Guide is dedicated to explaining those
+details.
+
+🄳 **[Stock options](https://en.wikipedia.org/wiki/Employee_stock_option)** are contracts that
+allow you to buy shares in the company you work for at a certain price.
+More formally, they are known as **employee stock options (or ESOs)**. When you buy shares,
```

Comment:
Not necessarily more "formal," but certainly more correct. The new clarity following is nicely done.